distributor-node/src/commands/leader/set-node-operational-status.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  INodeOperationalStatusMetadata,
+  ISetNodeOperationalStatus,
+  NodeOperationalStatusMetadata,
+  SetNodeOperationalStatus,
+} from '@joystream/metadata-protobuf'
+import AccountsCommandBase from '../../command-base/accounts'
+import DefaultCommandBase, { flags } from '../../command-base/default'
+import { NODE_OPERATIONAL_STATUS_OPTIONS, NodeOperationalStatus } from '../../types/metadata'
+
+export default class LeadSetNodeOperationalStatus extends AccountsCommandBase {
+  static description = `Set/update distribution node operational status. Requires distribution working group leader permissions.`
+
+  static flags = {
+    bucketId: flags.bucketId({
+      required: true,
+    }),
+    workerId: flags.integer({
+      char: 'w',
+      description: 'ID of the operator (distribution group worker)',
+      required: true,
+    }),
+    operationalStatus: flags.enum<NodeOperationalStatus>({
+      char: 'o',
+      options: [...NODE_OPERATIONAL_STATUS_OPTIONS],
+      required: false,
+      description: 'Operational status of the operator',
+    }),
+    ...DefaultCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { bucketId, workerId, operationalStatus: statusType } = this.parse(LeadSetNodeOperationalStatus).flags
+    const leadKey = await this.getDistributorLeadKey()
+
+    let operationalStatus: INodeOperationalStatusMetadata = {}
+    switch (statusType) {
```

Comment:
`statusType` input is not required, should we default to `Normal`?

Reply:
> It makes more sense to make it required, the command name doesn't imply what the default value should be.

Done